### ISSUE TITLE
Mix crash on startup

### ISSUE THREAD

**Issue by JohnDorien1** [ethereum/mix#251]
Mix sometimes freezes on startup. Ubuntu 14.04. Error in console:

```
Waiting for current execution to complete
/build/cpp-ethereum-jiMJrn/cpp-ethereum-1.1.4-SNAPSHOT-189-20160218-413f624~trusty/mix/src/CodeModel.cpp(359): Throw in function dev::mix::CompiledContract* dev::mix::CodeModel::contract(const QString&)
Dynamic exception type: boost::exception_detail::clone_impl<dev::Exception>
std::exception::what: std::exception
[dev::tag_comment*] = Contract not found: Sample
```

Looks like an issue when contract "sample" is not available anymore, but the tx on it is still existant in scenario

**Comment by yann300** [comment]
if you select the transaction and press "suppr". it should be removed from the scenario.

**Comment by JohnDorien1** [comment]
Unfortunately I cannot select & press anything as mix is frozen more or less instantly. Removing the .mix file to start off a fresh project does not help either, as mix still freezes then (see other issue). Removing the Sample Tx manually from the .mix file leads to same error message as having .mix file deleted. Result is, Mix is completely unusable for me now

**Comment by LianaHus** [comment]
from gitter chat: 
JohnDorien1:
so actually, restoring the deleted .mix leads to issue #251, having it deleted leads to the immediate freeze upon opening a project or creating a new one.

**Comment by yann300** [comment]
should be fixed when #252 will be addressed
